src/core/vdom.ts

```diff
@@ -323,8 +332,12 @@ export function w<W extends WidgetBaseTypes>(
 		| LazyDefine<W>
 		| Callback<any, any, RenderResult>,
 	properties: W['properties'],
-	children?: W['children']
+	children?: any
 ): WNode<W> {
+	if ((properties as any).children) {
```

Comment:
is this correct now? would it not be `__children__`?

Reply:
Nice catch

Reply:
Fixed and a couple of other misses too.